.github/workflows/docker-build.yml

```
@@ -4,10 +4,11 @@ on:
   push:
     branches:
       - "**"
```

This should be main, then you don't need to have the `github.ref` check down below. 
```suggestion
      - main
```

---

testing just `push` event due to the `if` condition filtering